File: Baekjoon_Online_Judge/NO_14264/14264.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 20                   | #
# | 14264 정육각형과 삼각형     | #
# ---------------------------- #


if __name__ == "__main__":
    L = int(r_input())
    print(((3 * 3 ** (1 / 2) * L ** 2) / 2) / 6)

File: Baekjoon_Online_Judge/NO_14264/14264.cpp
#include <iostream>
#include "math.h"
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 20                   | 
 | 14264 정육각형과 삼각형     | 
 ---------------------------- 
*/

int main(){
    int L;
    cin >> L;
    cout.precision(12);
    cout << ((3 * sqrt(3) * pow(L, 2)) / 2) / 6;
    return 0;
}
